clang/test/SemaSYCL/sycl-device-intel-reqd-work-group-size-template.cpp

```diff
@@ -84,3 +84,18 @@ int check() {
 // CHECK-NEXT: SubstNonTypeTemplateParmExpr {{.*}}
 // CHECK-NEXT: NonTypeTemplateParmDecl {{.*}}
 // CHECK-NEXT: IntegerLiteral{{.*}}8{{$}}
+
+// No diagnostic is emitted because the arguments match. Duplicate attribute is silently ignored.
+[[sycl::reqd_work_group_size(4, 4, 4)]] [[sycl::reqd_work_group_size(4, 4, 4)]] void func4() {}
```

Comment:
We should not push new symmetrical examples/test like this.
Any mixing X, Y and Z would just pass the test.